### PR TITLE
Flatpak fix flathub error

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
This adds the name for the community for the flatpak. It's needed in order for the flatpak to run.

This is where the error message is from.

https://buildbot.flathub.org/#/builders/31/builds/10616

### CODE REVIEW

File: flatpak/org.feichtmeier.Musicpod.appdata.xml [hunk 1]
```diff
@@ -14,7 +14,9 @@
       Play local audio files, browse podcasts online or listen to internet radio stations.
     </p>
   </description>
-  
+  <developer>
```
Comment (bbhtt) [inline flatpak/org.feichtmeier.Musicpod.appdata.xml:17]: needs id https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#name-summary-and-developer-name

Comment (TheShadowOfHassen) [inline flatpak/org.feichtmeier.Musicpod.appdata.xml:17]:
```suggestion
  <developer id="org.ubuntufluttercommunity">
```

Comment (TheShadowOfHassen) [reply flatpak/org.feichtmeier.Musicpod.appdata.xml]: Luckily, Ubuntu Flutter Community owns ubuntufluttercommunity.org, so I used that. I'm assuming that'll work, right?

Comment (bbhtt) [reply flatpak/org.feichtmeier.Musicpod.appdata.xml]: Yea anything would work, neither Appstream nor Flathub checks the validity of the id here.

**Comment by Feichtmeier** [comment]
I bought feichtmeier.org just for this lovely flatpak. Now please use it :)

File: flatpak/org.feichtmeier.Musicpod.appdata.xml [hunk 2]
```diff
@@ -14,7 +14,9 @@
       Play local audio files, browse podcasts online or listen to internet radio stations.
     </p>
   </description>
-  
+  <developer id="org.ubuntufluttercommunity">
```
Comment (Feichtmeier) [inline flatpak/org.feichtmeier.Musicpod.appdata.xml:17]: <developer id="org.feichtmeier">
  <name>Frederik Feichtmeier</name>
</developer>

**Review by Feichtmeier — CHANGES_REQUESTED** [review]

File: flatpak/org.feichtmeier.Musicpod.appdata.xml [hunk 3]
```diff
@@ -14,7 +14,9 @@
...
+    <name>Ubuntu Flutter Comunity</name>
```
Comment (TheShadowOfHassen) [inline flatpak/org.feichtmeier.Musicpod.appdata.xml:18]:
```suggestion
```

**Comment by TheShadowOfHassen** [comment]
How's that?

**Pull request merged**